packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -804,9 +806,27 @@ export default class Meetings extends WebexPlugin {
       );
 
       this.meetingCollection.set(meeting);
+      const isMeetingActive = destination.fullState.active;
+      const handleLocusMercuryIsSource = (source === 'handleLocusMercury');
```

Comment:
Just a suggestion rather then having the function name as a key to decide if we need time cant we just pass a boolean saying useRandomDelayForInfo or something name 

any thoughts !?